neptune/new/constants.py

```diff
@@ -31,3 +31,7 @@
 ASYNC_DIRECTORY = "async"
 
 OFFLINE_NAME_PREFIX = "offline/"
+CONTAINER_DIR_REGEXP = (
```

Comment:
Really not sure about usefulness of this. Couldn't we just `split("__")` and be done with it?

Reply:
But it's used in other places now.
It's a nice way to rely on regexp group to determine run UUID since `<type>__` prefix is optional.

Reply:
Actually, it's not nice. There is no contract that container IDs are/contain UUID AFAIK. The format we use is "<type>__<unique_id>" and that's what the should check for IMO (again, `split("__")`?)

Reply:
Ok